android-auto-app/src/main/java/com/mapbox/maps/testapp/auto/car/MapSession.kt

```diff
@@ -16,9 +16,10 @@ import com.mapbox.maps.extension.androidauto.mapboxMapInstaller
  */
 @OptIn(MapboxExperimental::class)
 class MapSession : Session() {
+
   private val carMapShowcase = CarMapShowcase()
   private val mapboxCarMap = mapboxMapInstaller()
-    .onCreated(CarMapWidgets(), carMapShowcase)
+    .onCreated(CarAnimationThreadController(), CarMapWidgets(), carMapShowcase)
```

Comment:
Note that this is not part of the installer, because Android Automotive will not have this issue

Reply:
Although, maybe it should go inside anyways 🤔

Reply:
I can't advise anything here but in general better not use background thread unless there's absolute need for that imo

Reply:
think it's good like this for now. it's easy to add but not as easy to remove. resolving here